skymp5-server/ts/settings.ts

```diff
@@ -32,6 +39,7 @@ export class Settings {
         "gamemodePath",
         "loadOrder",
         "dataDir",
+        'startPoints',
```

Comment:
Please use double quotes (" ") here instead of single quotes (' ').
It's the last one and I am going to merge this PR.